CONTRIBUTING.md

```diff
@@ -21,13 +21,10 @@ All code changes happen through pull requests. They are the best way to propose
 We actively welcome your pull requests:
 
 1. Fork the repo and create your branch from `master`.
-2. If you've added code that should be tested, add tests.
-3. If you've changed APIs, update the documentation.
-4. Ensure the test suite passes.
-5. Make sure your code lints.
-6. Issue that pull request!
-7. Review notes from team and update PR accordingly
-8. Once PR has been approved, the branch is deleted.
+2. Issue that pull request!
+3. Review notes from team and update PR accordingly
+4. Once PR has been approved, the branch is deleted.
+5. For more info see the [Github Flow](https://guides.github.com/introduction/flow/index.html).
```

Comment:
just a note, in markdown you can use `1.` for every item and the renderer will sort it out